ironfish/src/rpc/routes/accounts/getTransaction.ts

```diff
@@ -3,26 +3,25 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 import * as yup from 'yup'
 import { ApiNamespace, router } from '../router'
-import { getAccount, getTransactionNotes, getTransactionStatus } from './utils'
+import { serializeRpcAccountDecryptedNote, serializeRpcAccountTransaction } from './types'
+import { getAccount, getTransactionStatus } from './utils'
 
 export type GetAccountTransactionRequest = { account?: string; hash: string }
```

Comment:
Nit, I noticed this request it has name `hash`, but in the response we referred it as `transactionHash`. It will be good if we keep the naming consistent over our API schema

Reply:
More specifically, in Mira's PR https://github.com/iron-fish/ironfish/pull/2246/files
it is named `transactionHash`. Can we decide on one name and stick with it moving forward.

Reply:
I think we should stick with this name then, as if you're in an api called GetX, you should be able to know the hash is to identify X. Any other hash is should be clarified.